core/wire/commands/old_client_commands.go

```diff
@@ -138,7 +162,11 @@ func (c *MessageEmpty) ToBytes() []byte {
 	binary.BigEndian.PutUint32(out[2:6], uint32(c.Cmds.messageEmptyLength()))
 	out[6] = byte(messageTypeEmpty)
 	binary.BigEndian.PutUint32(out[8:12], c.Sequence)
-	return padToMaxCommandSize(out, c.Cmds.maxMessageLen(c))
+	return c.Cmds.padToMaxCommandSize(out, false)
+}
+
+func (c *MessageEmpty) Length() int {
+	return cmdOverhead + 4 + c.Cmds.geo.PacketLength
```

Comment:
Why is 4 special and if it is special, can it be given a constant variable name that ensures no one asks this question again?

Reply:
4 bytes in a uint32

Reply:
Okay, understood.